00_jit.ipynb

```diff
@@ -389,6 +284,68 @@
     "> As these are just the models, raw probabilities are returned. You still need to perform a soft or argmax"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You could also save a cpu version of the model:"
```

Comment:
Don't add cpu version example, specify the flag explicitly in the first example to be efficient.